### ISSUE TITLE
"Use of moved value" false positive

### ISSUE THREAD

**Issue by fujiapple852** [intellij-rust/intellij-rust#3840]
## Environment

* **Intellij-Rust plugin version:** v0.2.97.2123-191
* **Rust toolchain version:** v1.34.1
* **IDE name and version:** CLion 2019.1.2
* **Operating system:** OSX

## Problem description
The `main()` function below compiles (and is clean on Clippy) but is incorrectly flagged as a "Use of a moved value" on the `now` parameter to `bar()`.  Note that `DateTime` and `Utc` both implement the `Copy` trait.  

Switching to the explicit `add` method on `DateTime` avoids the false positive.

```rust
use chrono::{DateTime, Utc};
use time::Duration;

fn main() {
    let now: DateTime<Utc> = Utc::now();
    let soon = now + Duration::seconds(10);
    bar(now, soon);
}

fn bar(now: DateTime<Utc>, soon: DateTime<Utc>) {
    println!("now = {:?}, soon = {:?}", now, soon);
}
```

## Steps to reproduce
See: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=b0fb01a26c933c7e7144e41619c8443e

**Comment by vlad20012** [comment]
This seems to be foxed now
